syntax = "proto3";

package proto.ext.file_search;

message SearchRequest {
  string query = 1;
};

message FileInfo {
  string path = 1;
  string mime_type = 2;
};

message SearchResponse {
  repeated FileInfo files = 1;
};

message Request {
  oneof payload {
    SearchRequest search = 1;
  };
};

message Response {
  oneof payload {
    SearchResponse search = 1;
  }
};
